db/migrate/20160309144534_create_consoles.rb

```diff
@@ -0,0 +1,18 @@
+class CreateConsoles < ActiveRecord::Migration[5.0]
+  def change
+    create_table :consoles do |t|
+      t.string :protocol
+      t.string :host_name
+      t.integer :port
+      t.boolean :ssl
+      t.string :secret
```

Comment:
This is a one time password (valid for 2 minutes) for loggin in to the console. It can not be encrypted, because of [this](https://github.com/ManageIQ/manageiq/pull/7078/files#diff-973578637755c378be9d913af93a01e2R57) and [this](https://github.com/ManageIQ/manageiq/pull/7078/files#diff-4c8342ed474fde5dcb87be9ae7d96edaR13).